frontend/test/metabase/scenarios/question/nested.cy.spec.js

```diff
@@ -532,6 +534,27 @@ describe("scenarios > question > nested", () => {
       );
     });
   });
+
+  it("should be able to use integer filter on a nested query based on a saved native question (metabase#15808)", () => {
```

Comment:
In some places we have the `reproductions` dir inside a `scenarios` dir.

Here, the repro is directly in a file inside `scenarios` but not in a `reproductions` dir.

How do you decide which one goes where?